pyhms/sprout/sprout_filters.py

```diff
@@ -89,16 +87,14 @@ def __init__(self, limit: int) -> None:
 
     def __call__(
         self,
-        candidates: Dict[AbstractDeme, Tuple[Dict[str, float], List[Individual]]],
+        candidates: dict[AbstractDeme, DemeCandidates],
         _,
-    ) -> Dict[AbstractDeme, Tuple[Dict[str, float], List[Individual]]]:
+    ) -> dict[AbstractDeme, DemeCandidates]:
         for deme in candidates.keys():
-            candidates[deme][1].sort(key=lambda ind: ind.fitness)
-            if len(candidates[deme][1]) > self.limit:
-                candidates[deme] = (
-                    candidates[deme][0],
-                    candidates[deme][1][: self.limit],
-                )
+            if len(candidates[deme].individuals) > self.limit:
+                candidates[deme].individuals = sorted(candidates[deme].individuals, key=lambda ind: ind.fitness)[
```

Comment:
does it work for maximization? why use key? Individuals are naturally ordered by fitness